posts/2020-06-04-Rust-1.44.0.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "Announcing Rust 1.44.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.44.0. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.44.0 is as easy as:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.44.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1440-2020-06-04
+
+## What's in 1.44.0 stable
+
+Rust 1.44 is a smaller release, containing almost exclusively polish and a
+few new stabilized standard library APIs.
+
+`cargo tree` is a very old, popular extension to Cargo. Originally released
+in December of 2015, and with almost 50,000 downloads, [we have decided to
+add it to Cargo directly][cargotree]. If you haven't used it in the past,
+`cargo tree` will print your dependency tree to the console:
+
+```text
+  mdbook v0.3.2 (/Users/src/rust/mdbook)
+├── ammonia v3.0.0
+│   ├── html5ever v0.24.0
+│   │   ├── log v0.4.8
+│   │   │   └── cfg-if v0.1.9
+│   │   ├── mac v0.1.1
+│   │   └── markup5ever v0.9.0
+│   │       ├── log v0.4.8 (*)
+│   │       ├── phf v0.7.24
+│   │       │   └── phf_shared v0.7.24
+│   │       │       ├── siphasher v0.2.3
+│   │       │       └── unicase v1.4.2
+│   │       │           [build-dependencies]
+│   │       │           └── version_check v0.1.5
+...
+```
+
+In this example, the `mdbook` package depends on the `ammonia` package,
+which itself depends on `html5ever`, which depends on `log`, `mac`, and
+`markup5ever`... and so on.
+
```

Comment:
```suggestion

You can also use `cargo tree -d` (shorthand for `cargo tree --duplicates`) to show any dependencies on multiple versions of the same crate. This will show one entry for each version of each dependency, and an inverted dependency tree showing how that version gets pulled in.

```

I've avoided giving an example here, to avoid singling out any specific crates (which will likely get fixed in the future).